server/ColorDeconvolution/ColorDeconvolution.xml

```diff
@@ -60,5 +60,42 @@
       <longflag>stainColor_3</longflag>
       <default>0,0,0</default>
     </double-vector>
+    <boolean>
+      <name>largeImage</name>
+      <label>Large Image?</label>
+      <longflag>largeImage</longflag>
+      <description>True if the input image is a large image that should be handled with large_image</description>
+    </boolean>
+    <integer-vector>
+      <name>region</name>
+      <label>Region</label>
+      <longflag>region</longflag>
+      <description>left,top,width,height of the region of interest</description>
+    </integer-vector>
```

Comment:
Use the region type in Slicer XML Spec. See example [here](https://github.com/DigitalSlideArchive/HistomicsTK/blob/master/server/NucleiDetection/NucleiDetection.xml#L127-L133_